### ISSUE TITLE
Split screen virtual keyboard

### ISSUE THREAD

**Issue by cip** [cip/WikiOnBoard#50]
Implemented in:
https://github.com/cip/WikiOnBoard/commit/d734fd867bce582966cf77f74cc5d368fe9d4355
Issues: 
- Could only compile in Carbide/FP1+Qt.4.6.3
  (Only once, cannot compile anymore (Probably unrelated to this change)))
  => working fine, except that problem with resizing if switched from landscape to portrait with keyboard open.
- Creator S1+4.6.3 not working because symbian native libs not included in sdk.
- Creator S3 would include libs, but cannot compile due to unrelated problems. (and also not that good, because than for s1 extra compile necessary)

**Comment by cip** [comment]
Implemented 0.0.48
Removed all additional libs except fepbase => compiles on S^1+4.6.3 again
New issue created "Orientation change issue with split-screen keyboard"

**Comment by cip** [comment]
Consider removal of split input keyboard.
1. This issue is hard to fix. (Perhaps disabling/enabling on rotatation)
2. Will be supported by qt 4.7.4
   Not really, appearantly not added for Widgets ...
      http://bugreports.qt.nokia.com/browse/QTBUG-16572
